src/workingtitle-vcockpits-instruments-cj4/html_ui/Pages/VCockpit/Instruments/Airliners/CJ4/FMC/CJ4_FMC.js

```diff
@@ -38,6 +38,12 @@ class CJ4_FMC extends FMCMainDisplay {
         this._activatingDirectTo = false;
         this._templateRenderer = undefined;
         this._msg = "";
+		this.DSPLMENUcounter = 0;
```

Comment:
Insteaf of keeping track of these values in fmc, wouldn't we want to always work off of the Simvar? Could make it a getter property that always reads from simvar